src/main/java/com/conveyal/taui/controllers/AggregationAreaController.java

```diff
@@ -56,7 +60,16 @@
             .build();
     private static final FileItemFactory fileItemFactory = new DiskFileItemFactory();
 
-    public static AggregationArea createAggregationArea (Request req, Response res) throws Exception {
+    /**
+     * Create binary .grid files for aggregation (aka mask) areas, save them to S3, and persist their details.
+     * @param req Must include a shapefile on which the aggregation area(s) will be based. If request includes
+     *            "union=true", features will be merged to a single aggregation area with the provided name. If not,
```

Comment:
Do we need the union operation at all? This seems like something that would be done upstream in GIS in the very rare instances where it would be needed, and should not be done silently server-side in our system.

Reply:
Point taken about simplifying the code, but I'm inclined to keep the current union operation as an option.  Though the aggregation area upload step isn't frequently used, some users are probably accustomed to the current behavior.  I think making them perform the union, add a property, and specify the name of the single unioned feature upstream in GIS would be annoying compared to the current single-step process.